### ISSUE TITLE
Test debug window not disappearing

### ISSUE THREAD

**Issue by R4Z3R3X** [thedigicraft/Atom.CMS#72]
Hey, when i refreshed the page after typing the jquery script, the message at the bottom "Test debug window" is not disappearing like it has for you. I copied exactly as you have typed it. Whats going on?

I am referring to part 17 in the AtomCMS 2.0 series.
Any help would be much appreciated, also love the videos, thanks very much.

``````
<script>

    $(document).ready(function() {

        $("#console-debug").hide();

    });
    ```
</script>
``````

**Comment by marklaffan** [comment]
R4ZERX420, hi, I have the same problem you had but I just dont understand how your solution is put into practice, any help please?

**Comment by marklaffan** [comment]
Got it!! Sorry

**Comment by Lidosurf** [comment]
This worked Thanks

**Comment by ozyedi** [comment]
i dont understand where is the problem, here is my jquery versions;

<script src"https://code.jquery.com/jquery-3.0.0.min.js"></script>

<script src="https://code.jquery.com/ui/1.12.0-rc.2/jquery-ui.min.js"></script>

<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>

R4ZERX420 says something about versions, can anyone tell me where is the problem?
many thanxs already

**Comment by creptor** [comment]
It basically depends on what you're doing. Each Jquery version offers the same or better functionalities but with minor changes to variables and other things.

For this page version you should use Jquery v2.x or a 2.x with migrate... you can get these from our friend Google at https://developers.google.com/speed/libraries/#jquery .

**Comment by creptor** [comment]
Note: using an outdated version of Jquery could give errors to other JavaScripts that use it has library. To avoid this try to get the version that all your JavaScript supports.

**Comment by ozyedi** [comment]
i've changed CDN's to newest stable ones, and its done.
Thnx

**Comment by v917** [comment]
My test debug window is coming on top of the footer. I had also put its position: absolute and once position: relative but nothing happened. The jquery hiding feature is working perfectly but this position of debug window is messing all this. Please Help. Thanks in advance.

**Comment by creptor** [comment]
Please create a new issue with your problem, and add the code using https://gist.github.com. I'll try to help you fix it.

**Comment by Chriswilldo** [comment]
I hate when this happens!!! If you want this to work as precisely as his, don't worry about trying to add in your own flavor into the mix until the mixture is fully finished. Believe ME!!! IT'S NOT WORTH the initial effort. I've done this 7 times over. Mainly due to lack of consistent commitment and lost work. For your issue, don't worry too much on it. I've had THIS happen to me almost every time. I think I was able to get it below my footer on my 1st, 3rd, and 6th attempt, but overall it still doesn't really matter if you scan over the next few videos. This is a 112 part series need I remind you. Try not to get to anal about little spits like this. It will deteriorate you!!! If you get far enough, you'll see some of the errors I've needed help on. Some of them were simply cuz I wanted to try to implement some alternate codes. If you want this to work EXACTLY like his, make sure your aptana, xampp, jquery; ui, js, bootstrap, etc... coding is exactly as his! If you try to use the newer versions as I have, if you aren't 110% as accurate as he is then you WILL run into errors. And I meant EXACT!!!! IF YOU MAKE ANY ERRORS WHILE TYPING YOUR CODE AND FIX IT MID LINE, DO UNDERSTAND THAT THERE'S A GOOD CHANCE IT WON'T WORK. Every character must be precise. ESPECIALLY ONCE YOU GET INTO $Q = "SELECT * FROM" , $r = mysqli_result. This is my biggest downfall I promise you! I can't tell you how many times I had to retype this flunking code just to get it to work! Often times I'd need CREPTOR'S help just to get THIS code functioning again.. smh
